frontend/src/metabase/plugins/index.ts

```diff
@@ -19,13 +19,11 @@ import type {
   User,
 } from "metabase-types/api";
 import type { AdminPathKey, State } from "metabase-types/store";
+import type { TimelineEvent } from "metabase/components/Timeline";
```

Comment:
Could we move this type here, in `plugins/index.ts` instead of using one from the component? Components should depend on plugins but not wise-versa. And another thing - we already have an entity called `TimelineEvent`, could we name this one differently? Like `RevisionEvent` or whatever

Reply:
Yup I can do those both